.plzconfig.ci

```diff
@@ -5,3 +5,8 @@ dircompress = true
 
 [buildconfig]
 ci = true
+
+; CI should disable all feature flags so we're actually testing the default distribution.
```

Comment:
I'm a little lost about why we're enabling it in the main config but disabling here. Won't that mean it is effectively disabled everywhere since all our PRs will have to pass CI?

Reply:
Well I think the CI stuff should test the code our users are likely to run so should definitely have it disabled by default. It's enabled in the main config so when I'm testing locally, I'm testing my latest changes. 

Ideally we'd have one CI platform (linux-alt?) that has them enabled so CI tests both.